src/components/curriculum-section/curriculum-info-section/CurriculumInfoSection.js

```diff
@@ -28,46 +27,26 @@ const CurriculumInfoSection = ({ curriculumId }) => {
   if (isErrorCurriculum || isErrorCurriculumCycles) {
     return 'error';
   }
-  const paragraphsArray = curriculum?.description?.split('\\n');
-  const CurriculumDescription = () => {
-    return (
-      <>
-        <Head>
-          <title>{curriculum.name}</title>
-          <meta property="og:title" content={curriculum.name} key="title" />
-        </Head>
-        <div className={style.flex_item}>
-          {paragraphsArray?.map((paragraph) => (
-            <p key={`paragraph-${paragraphsArray.indexOf(paragraph)}`}>
-              {paragraph}
-            </p>
-          )) ?? null}
-          {'\n'}
-          <i>
-            Explora los ciclos de desarrollo y sus objetivos de aprendizaje más
-            abajo.
-          </i>
-        </div>
-      </>
-    );
-  };
 
   return (
-    <>
-      <Panel
-        header=<CurriculumHeader curriculumName={curriculum?.name ?? null} />
-      >
-        <div className={style.flex_container}>
-          <Image
-            className={`${style.curriculum_image} ${style.flex_item}`}
-            src={person_looking_curriculum}
-            alt="drawed person looking curriculum"
-          />
-          <CurriculumDescription />
-        </div>
-        <CyclesSection cycles={cycles} />
-      </Panel>
-    </>
+    <Panel header={curriculum.name}>
+      <div className={style.row}>
+        <Card className={style.leftColumn}>
+          <div className={style.row}>
+            <Image
+              src={laptopImage}
+              alt="img_curriculum"
+              width="330"
+              height="220"
+            />
+            <p className={style.paragrapText}>{curriculum.description}</p>
```

Comment:
keep in mind that if we're showing the whole description as just one paragraph we must update the seeds to remove the \n that we have there